### ISSUE TITLE
libxml2 is hard to install.

### ISSUE THREAD

**Issue by pc10201** [retspen/webvirtcloud#123]
import libxml2
has error.

I spend a lot time to install it.
You can use lxml to replace libxml2.

**Comment by master2sid2** [comment]
Just replce in /srv/webvirtcloud/conf/requirements.txt from 
http://git.gnome.org/browse/libxml2/snapshot/libxml2-2.9.1.tar.gz#egg=libxml2-python&subdirectory=python
to
http://github.com/retspen/retspen.github.io/raw/master/libxml2-python-2.9.1.tar.gz

**Comment by ikus060** [comment]
Any reason, why this changed is not in the repo already ?

**Comment by hongweipeng** [comment]
yeah, I run this in Python3.  There is not libxml2 for Python3. So I use lxml replace.
